docker/release/nanoserver/Dockerfile

```diff
@@ -0,0 +1,59 @@
+# escape=`
```

Comment:
As far as I see, that causes to create a single line for the whole PS code. It's a little bit tedious to have a end-of-line escaping on every line. Even in vanilla powershell it's common to prefer **splatting** over end-of-line escaping, because it's more  maintainable. What happens, if you just remove the backticks from the whole file?